python/lsst/dax/imgserv/api_soda.py

```diff
@@ -88,67 +88,81 @@ def load_imgserv_config(config_path=None, metaserv_url=None):
     # configure the log file (log4cxx)
     log.configure(os.path.join(config_path, "log.properties"))
     current_app.config["DAX_IMG_CONFIG"] = config_path
-    if metaserv_url is not None:
-        current_app.config["dax.imgserv.meta.url"] = metaserv_url
+    if metaserv_url:
+        current_app.config["DAX_IMG_META_URL"] = metaserv_url
     current_app.config["imgserv_api"]=os.path.join(config_path,
                                                    "image_api_schema.json")
     # create cache for butler instances
     current_app.butler_instances = {}
     # create SODA service
-    f_dashboard = os.path.join(config_path, "dashboard.json")
-    with open(f_dashboard, "r") as f:
-        dashboard = json.load(f)
-    current_app.soda = ImageSODA(current_app.config, dashboard)
+    current_app.soda = ImageSODA(current_app.config)
 
 
 @image_soda.route("/")
-def index():
+def is_index():
     return make_response(render_template("api_image_soda.html"))
 
 
 @image_soda.route("/availability", methods=["GET"])
-def imagesoda_availability():
+def is_availability():
```

Comment:
Why are you changing the names from imagesoda_ to is_?  Usually when I see a function that starts with is, it means it is a function that return a boolean.  Like, is_true, is_connected, etc.  I don't think that's what you're doing here?  I think getting rid of the is_ part and just call it availability, capabilities, etc, would be better

Reply:
Agree. Let me change the prefix to `img` instead.  Started out with naming as suggested by blocked by 'async' is a reserved keyword.